hawkular-rest-status/pom.xml

```diff
@@ -47,6 +47,11 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>org.hawkular.commons</groupId>
+      <artifactId>hawkular-tenant-jaxrs-filter</artifactId>
+      <version>0.7.3.Final-SNAPSHOT</version>
```

Comment:
While it might not be the best Maven practice (I honestly don't know and believe your expertise here), it's been used in Hawkular all over the place:
`find -name 'pom.xml' | xargs grep '${project.version}' | wc -l` in various components:

metrics: 42
inventory: 60
alerts: 33